File: Source/TurnTracker.Server/Models/ActivitySummary.cs
using System;

namespace TurnTracker.Server.Models
{
    public class ActivitySummary
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public int? CurrentTurnUserId { get; set; }
        public string CurrentTurnUserDisplayName { get; set; }
        public DateTimeOffset? Due { get; set; }
        public bool IsDisabled { get; set; }
    }
}

File: Source/TurnTracker.Client/src/app/user/models/ActivitySummary.ts
import { DateTime } from 'luxon';

export class ActivitySummary {
  public id: number;
  public name: string;
  public description: string;
  public currentTurnUserId?: number;
  public currentTurnUserDisplayName?: string;
  public due?: string;
  public isDisabled: boolean;

  // Properties populated by the client and not the server
  public dueDate?: DateTime;
  public overdue: boolean;
}
